src/ex2/app.js

```diff
@@ -0,0 +1,124 @@
+import { ItemManager } from "./itemManager.js"
+import { PokemonClient } from "./pokemonClient.js"
+
+const addButton = document.getElementById("list-item-submit")
+const textInput = document.getElementById("list-item-input")
+const allLists = document.getElementById("list")
+const counterLists = document.getElementById("counter")
+const deleteAllLists = document.getElementById("deleteAllTasks")
+const errorMessage = document.getElementById("error")
+const regex = /^[0-9]*$/
+
+const itemManager = new ItemManager()
+const pokemonClient = new PokemonClient()
```

Comment:
you already declared main class and initiate it on dom ready, you should move this to the constructor and all of your other functions into the class itself